classes/controllers/grid/users/reviewer/PKPReviewerGridHandler.inc.php

```diff
@@ -531,6 +531,8 @@ function readReview($args, $request) {
 			$templateMgr->assign('commentPrivate', $submissionCommentPrivate?$submissionCommentPrivate->getComments():'');
 		}
 
+		$templateMgr->assign_by_ref('reviewerRecommendationOptions', ReviewAssignment::getReviewerRecommendationOptions());
```

Comment:
(but I merged it anyway, as that's very nitpicky and I didn't have other comments!)